backend/src/main/java/com/tourneynizer/tourneynizer/controller/TeamRequestController.java

```diff
@@ -109,11 +109,11 @@ public TeamRequestController(TeamRequestService teamRequestService, SessionServi
     }
 
     @GetMapping("/api/user/requests/pending")
-    public ResponseEntity<?> getRequestsByUser(@CookieValue("session") String session) {
+    public ResponseEntity<?> getRequestsForUser(@CookieValue("session") String session) {
```

Comment:
Since you switched the urls for by and for here why didn't you do it for the teams as well?

Reply:
Done.